viewing_party/party.py

```diff
@@ -0,0 +1,208 @@
+# test_wave_01
+def create_movie(movie_title, genre, rating):
+    movie={}
+    if movie_title == None or genre == None or rating ==None :
```

Comment:
The description for this function says to return `None` if any of the arguments are falsy. The current implementation explicitly checks for None (which is a very common falsy value), but we could better match the specification with a condition like this:
```suggestion
    if not movie_title or not genre or not rating:
```